src/components/ae-overscreen/ae-overscreen.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="ae-overscreen" :class="[{ shadow }, align]">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ae-overscreen',
+  props: {
+    /**
+     * Applies a linear shadow gradient to the overscreen
+     */
+    shadow: Boolean,
+
+    /**
+     * Align the elements either left or right, default alignment is center.
+     * Available values are: `left, right`
+     */
+    align: {
+      type: String,
+      validator: value => [
+        'left',
+        'center',
+        'right',
+      ].includes(value),
+      default: 'center',
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../styles/globals';
+
+.ae-overscreen {
+  display: flex;
+  align-items: flex-end;
```

Comment:
Is this property actually necessary?

Reply:
yes.

Reply:
In what case?

Reply:
The default style behavior of this component, should align items to the bottom!